### ISSUE TITLE
An exception occurred processing JSP page /WEB-INF/jsp/selectDistrict.jsp at line 40

### ISSUE THREAD

**Issue by smithleej** [kittylyst/VillagePaymentAgent#47]
This issue is related to #34 
When on Verifier Registration after going past selecting the country, when you click next to select the region you, receive a page with the following message:
```
Caused by:

org.apache.jasper.JasperException: An exception occurred processing JSP page /WEB-INF/jsp/selectDistrict.jsp at line 40

37: 
38:                        <td>District : </td>
39:                        <td><form:select path="idDistrict">
40:                             <form:options items="${districtList}" itemValue="id" itemLabel="description" />
41:                         </form:select></td>
42:                    </tr>
43:                    <tr>
```
The relevant section of the stacktrace is:

Caused by: org.hibernate.LazyInitializationException: could not initialize proxy - no Session
	at org.hibernate.proxy.AbstractLazyInitializer.initialize(AbstractLazyInitializer.java:167)
	at org.hibernate.proxy.AbstractLazyInitializer.getImplementation(AbstractLazyInitializer.java:215)
	at org.hibernate.proxy.pojo.javassist.JavassistLazyInitializer.invoke(JavassistLazyInitializer.java:190)
	at org.haftrust.verifier.model.Region_$$_javassist_1.toString(Region_$$_javassist_1.java)
	at java.lang.String.valueOf(String.java:2979)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at org.haftrust.verifier.model.District.toString(District.java:52)
	at org.springframework.util.ObjectUtils.nullSafeToString(ObjectUtils.java:594)
	at org.springframework.util.ObjectUtils.getDisplayString(ObjectUtils.java:540)

**Comment by smithleej** [comment]
I'll submit a similar fix as for #34

**Comment by smithleej** [comment]
Fixed by pull request #48

**Comment by smithleej** [comment]
Fixed by pull request #48

**Comment by smithleej** [comment]
Fixed by pull request #48
